### DISCUSSION TITLE
Duplicate logEvents

### DISCUSSION THREAD

**Discussion by swdavidelias** [oddbit/flutter_facebook_app_events#319] [Q&A]
All targeted logEvent sends are being duplicated, with each new send it creates two of the same event.

![Captura de Tela 2023-10-25 às 11 22 17](https://github.com/oddbit/flutter_facebook_app_events/assets/128707992/2d8a9fdd-dc78-417c-80a6-5adde64b5ed4)

**Comment by luisgustavosc** [comment]
I'm experiencing the same behavior, even with automatically logged events as "Purchase". did you figure it out? @swdavidelias
